posts/2019-01-17-Rust-1.32.0.md

```diff
@@ -0,0 +1,274 @@
+---
+layout: post
+title: "Announcing Rust 1.32.0"
+author: The Rust Release Team
+---
+
+The Rust team is happy to announce a new version of Rust, 1.32.0. Rust is a
+programming language that is empowering everyone to build reliable and
+efficient software.
+
+If you have a previous version of Rust installed via rustup, getting Rust
+1.32.0 is as easy as:
+
+```
+$ rustup update stable
+```
+
+If you don't have it already, you can [get `rustup`][install] from the
+appropriate page on our website, and check out the [detailed release notes for
+1.32.0][notes] on GitHub.
+
+[install]: https://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/master/RELEASES.md#version-1320-2019-01-17
+
+## What's in 1.32.0 stable
+
+Rust 1.32.0 has a few quality of life improvements, switches the default
+allocator, and makes additional functions `const`. Read on for a few
+highlights, or see the [detailed release notes][notes] for additional
+information.
+
+#### The `dbg` macro
+
+First up, a small quality of life improvement. Are you a "printf debugger"? If you are, and
```

Comment:
@ashleygwilliams On the one hand, it's great to know that there are people out there who have used `println!` but never `printf`!

...on the other hand, "printf debugger" is in quotes here because it's actually [a relatively common phrase](https://www.google.com/search?q=%22printf+debugger%22) meaning "someone who uses print statements to debug their code".

Reply:
Actually, even more common is ["print debugger"](https://www.google.com/search?q=%22print+debugger%22), which makes no assumptions at all about your chosen language's IO story. So that's probably preferable.

Reply:
hey yeah i come from JS land, we call em "console.log debuggers"- think "print debugger" is great <3 (also i am a print debugger)